modules/ROOT/pages/update-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -0,0 +1,86 @@
+. If you wish to upload a new version, click on *Add new version* on the right-side table.
+. If you wish to change the tags of a version, you can do it in the *Tags* section on the right-side panel.  
+
+=== Update API instance in API Manager
+To update an existing API instance, perform the following steps:
+
+. Go to *Anypoint Platform > API Manager*
+. Search and select your API instance in the table. 
++
+Once inside the API details page, you can perform the following actions:
+. Update the instance tags, at the *API Configuration* section.
+. Update the asset version it points to.
+image::update-api-asset.png[Display asset update]
+
+
+[[update-api-cli]]
+== Update an API using the ASM CLI
+
+When you configured the API using the Anypoint Service Mesh autocreate feature, you can update it from the cluster as well. The update will reuse the environment information and credentials introduced in the provisioning. 
+
+To update, perform the following steps:
+
+. Using Anypoint Service Mesh, type the following command to create the API:
++
+[source,text,linenums]
+----
+asmctl api update  
+--name=_Kubernetes API resource name_ \
+--namespace=_namespace name_ \
+--version=_(Optional) new asset version for the API_ \
+--mainFile=_(Optional) name of the main file in the API specification_ \
+--specPath=_(Optional) path to the API specification file_ \
+--specTags=_list of comma separated tags to be added to the asset on Exchange_ \
+--instanceTags=_list of comma separated tags to be added to the instance on API Manager_ 
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required or Optional | Default Value
+| `name` | The name of the existing Kubernetes API resource, for example, `orders-api` | Required | Not applicable
+| `namespace` | The namespace in which you created you API resource, for example, `mulesoft-apps` | Required | Not applicable
+| `version` | The version of the new asset on Exchange, including the semantic versioning format for major, minor, and patch releases  | Optional | Not applicable
+| `mainFile` | The name of the main file in the API specification file, compliant with the requirement that YAML or JSON files are named the same as the specification file, while the mainFile should match the primary file name when working with ZIP files | Required for `oas`, `raml` or `wsdl` APIs if `version` field is set | Not applicable
+| `specPath` | The relative or absolute path to the API specification file | Required for `oas`, `raml` or `wsdl` APIs if `version` field is set | Not applicable
+| `specTags` | The tags to add to the asset on Exchange |  Optional | Not applicable
+| `instanceTags` | The tags to add to the API instance on API Manager |  Optional | Not applicable
+|===
++
+. Verify the API status:
+`asmctl api list`
++
+The new API is shown as `Ready`.
+. Optionally, verify the changes in the Exchange asset and in the API instance in API Manager.
+
+== See Also
```

Comment:
Should we put that you can only bump minor, patch versions?

Reply:
I thought I had add it somewhere 😅 .
Now it is included in the sentence at the beginning of the page.

Reply:
Thanks :)